src/main/scala/de/retest/guistatemachine/api/SutStateIdentifier.scala

```diff
@@ -3,6 +3,6 @@ package de.retest.guistatemachine.api
 import de.retest.recheck.ui.descriptors.SutState
 
 class SutStateIdentifier(sutState: SutState) extends HashIdentifier(sutState) {
-  val msg = s"SutStateIdentifier[sutState=${sutState.toString}, hash=$hash]"
+  val msg = s"[sutState=${sutState.toString}, hash=${hash.substring(0, 8)}]"
```

Comment:
Good idea to shorten the string, this will be sufficient for visualization.

However, two things:

* IMHO `ActionIdentifier` should follow the same format as both are `HashIdentifiers`.
* I believe that `SutState#toString()` is useless, because it's almost always like `descriptor=[html]`. (It was there before, so not related to this PR.)

Reply:
![Screen Shot 2020-02-29 at 29 02 20  13 23 52](https://user-images.githubusercontent.com/1871610/75607376-c1e9c880-5af6-11ea-9790-ba2460ceea3d.png)
With various changes in different projects, this finally looks like this.